internal/services/users/user_resource.go

```diff
@@ -133,6 +133,25 @@ func userResource() *schema.Resource {
 				ValidateFunc: validation.StringLenBetween(0, 16),
 			},
 
+			"employee_type": {
+				Description:  "Captures enterprise worker type. For example, Employee, Contractor, Consultant, or Vendor.",
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"Employee", "Contractor", "Consultant", "Vendor"}, false),
+			},
+
+			"cost_center": {
+				Description: "The cost center associated with the user.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+
+			"division": {
+				Description: "The name of the division in which the user works.",
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+
```

Comment:
As above, could we order these fields so that all the Computed fields are first, then the Optional ones, and then the Computed ones, and each section in alphanumeric order?

Reply:
reordered - hopefully :-)